app/javascript/react/components/ReviewFormContainer.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+
+const ReviewTileContainer = (props) => {
+  const [review, setReview] = useState({})
+    const getReviews = async () => {
+      const reviewId = props.match.params.id
+      binding.pry
+      try {
+        const response = await fetch(`/api/v1/sauces/${reviewId}`)
+        if (!response.ok) {
+          const errorMessage = `${response.status} ${response.statusText}`
+          const error = new Error(errorMessage)
+          throw error 
+        }
+        const responseBody = await response.json()
+        setReview(responseBody)
+      } catch (error) {
+        console.log("error in fetch:", error)
+      }
+    }
+
+    useEffect(() => {
+      getReviews()
+    },[])
```

Comment:
you shouldnt have `getReviews` at all. Instead, you should serialize your reviews alongside of your Sauce. No getReviews!

Reply:
wait....ARE YOU MAKING A FETCH REQUST FOR THE REVIEWS THAT YOU ALREADY HAVE!?

Reply:
this entire component is confusing

Reply:
Deleted ReviewFormContainer